changelog.py

```diff
@@ -100,6 +100,14 @@ def write(self, f: TextIO):
                 url = contributor_url(name)
                 f.write(url)
 
+        if len(self.labels) > 0:
+            f.write("\n")
+            for label in self.labels:
+                section = self.sections[label]
+                for item in section.items:
+                    f.write("[#{0}]: https://github.com/intellij-rust/intellij-rust/pull/{0}\n"
+                            .format(item.pull_request_id))
```

Comment:
If some PR has more than one label it will be duplicate.
Also, maybe it makes sense to sort PRs?

Reply:
Fixed